src/App.jsx

```diff
@@ -1,3 +1,93 @@
-export const App = () => {
-  return <div>Find me in App.jsx!</div>;
-};
+// App.jsx
+
+import React, { useState, useEffect } from "react";
+import TaskForm from "./components/TaskForm";
+import TaskList from "./components/TaskList";
+import { useTaskStore } from "./components/store";
+import rewardsData from "./components/rewards.json"; // Importar el JSON
+import "./App.css";
+
+
+
+function App() {
+  const tasks = useTaskStore((state) => state.tasks);
+  const darkMode = useTaskStore((state) => state.darkMode);
+  const toggleDarkMode = useTaskStore((state) => state.toggleDarkMode);
+  const totalRewards = tasks.filter((task) => task.completed).length;
+
+  // Estado para almacenar las recompensas
+  const [rewards, setRewards] = useState([]);
+
+  // Cargar las recompensas desde rewards.json
+  useEffect(() => {
+    setRewards(rewardsData); // Carga inicial
+  }, []);
+
+  // Buscar la recompensa basada en el total de puntos
+  const currentReward = rewards.find((reward) => reward.points === totalRewards);
+
+
+  return (
+    <div
+      className={`container ${darkMode ? "dark-mode" : ""}`}
+      style={{
+        background: darkMode
+          ? `
+            linear-gradient(
+              rgba(0, 0, 0, 0.6), 
+              rgba(0, 0, 0, 0.6)
+            ),
+            url('/Background.jpg')`
+          : `url('/Background.jpg')`, // Imagen normal en light mode
+        backgroundSize: "cover",
+        backgroundPosition: "center",
+        backgroundRepeat: "no-repeat",
+        minHeight: "100vh",
+        padding: "20px",
+        color: darkMode ? "#fff" : "#000", // Cambiar color del texto
+      }}

```

Comment:
Try finding a way to keep the CSS to the CSS file since that's where you have the rest of your styling. Also, consider adding a shadow/blur of some sort because the text is hard to read.

Reply:
PS. On another note, I see you're still uploading your files instead of committing from the terminal. Have you actively chosen this or are you still unsure of how to do it after the session we had a couple of weeks back? Working with git from the terminal is really the developer way, and I'm sure you can do it if you want to.                                                        I Know how to do it but I prefer, if its possible, to work on local and upload the final version. This is because of my lack of time, I work mainly by nights and, sometimes, I am so tired that I prefer to save it on my local rather than to forget it and, do not save it and lost it (I have had this problem in the past).    Hope this will be allowed, if its possible.